public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -60,7 +60,7 @@ define(function(require) {
     if(publishUrl) {
       this.updateDialog(publishUrl, true);
     }
-
+/** TODO - not sure about this...
     if(publisher.fsync) {
```

Comment:
So that we can detect when to show the "You need to update your published project" dialog, which should be when a file was changed after a publish occurs.
We also need to do this for `fileRename` and `fileDelete` events